### ISSUE TITLE
[Bug]: Volume Adjustement doesnt work. 

### ISSUE THREAD

**Issue by Landixbtw** [prasanthrangan/hyprdots#823]
## Bug Report
I can't control the System Volume with either my Hardware (Keyboard Knob) or F11 / F12

### Description
I can't control my System Volume with my Keyboard Volume Knob nor the F11 / F12 Key, the animation pops up and the numbers change but there is no impact on the Volume. 

### Steps to Reproduce

1. Download Hyprland, with the Install script. 
2. Consume something with audio, and try to adjust the volume

### Expected Behavior

The Volume increases or decreases

### Actual Behavior

The Volume doesn't increase or decrease

### Screenshots

Cant screenshot it. 

[If applicable, add screenshots to help explain your problem.]

### Environment

[Please add any other relevant information about the bug and remove the unnecessary lines.]

e.g.,

- Linux Distro: [e.g., ArchLinux, EndeavourOS]
- Linux Version, output of `uname -a`:
- Hyprland Version, output of `hyprctl version`:
- Installation Environment: [e.g., On The Main Hardware, On A Virtual Machine]
- Other info: [e.g., Display Resolution, Resolution, etc]

### Additional Information

[Add any other information about the problem here. For example, you might include the error message, any recent changes that you made to the project, or any other relevant details.]

**Comment by kRHYME7** [comment]
I have the same issue when I increase/decrease the volume of a Bluetooth speaker. The Built in speaker on my laptop behaves normally.

**Comment by kRHYME7** [comment]
@Landixbtw  Did the problem occur again this time? recently updated my system and found an audio-related update I think pipewire. And after that, the issue was fixed on my end.

**Comment by Landixbtw** [comment]
As of right now, this is still a problem, but I will do a system update and then check again.

**Comment by Landixbtw** [comment]
Between 100% and 10% nothing is happening, at 5% volume, the sound gets quiter, and and 0% it mutes.

**Comment by kRHYME7** [comment]
I see, have you tried controlling the volume using the GUI PulseAudio? Maybe it can give as some lead 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/b6b3d2d0-8f2b-436f-8e44-66cbd5c42673)

**Comment by Landixbtw** [comment]
Same result as here ```Between 100% and 10% nothing is happening, at 5% volume, the sound gets quiter, and and 0% it mutes.```

**Comment by kRHYME7** [comment]
Indeed, not the volume script issue. I hope we can find an upstream issue related to this from pavucontol

**Comment by Landixbtw** [comment]
Works for me now, does it work for you ?

**Comment by kRHYME7** [comment]
Yes only happened once. and only happens on a  bluetooth device
 Can we close the issue now?
